### PR TITLE
Add limit on subjects uploaded from a user

### LINKED ISSUE #774: Hard limits for normal users?
Should we enforce limiting the number of projects a normal user can spin up? Also how many subjects and subject sets can they create for a project?

### PR DESCRIPTION
**Pull request by edpaget**
Adds a configuration file user_limits.yml with the form:
```
rails_env:
  subjects: 1000
```

Leaving it empty for an environment will cause it to not limit the
number of subjects.

Closes #774

### CODE REVIEW

File: config/user_limits.yml.hudson [hunk 1]
```diff
@@ -0,0 +1,5 @@
+development:
+
+test:
+  subjects: 100
+  projects: 100
```
Comment (camallen) [inline config/user_limits.yml.hudson]: do you want to remove projects since we're not limiting on it?

**Comment by camallen** [comment]
looks good to merge.

File: config/user_limits.yml.hudson (hunk 1 above)
Comment (edpaget) [inline config/user_limits.yml.hudson]: Ah yeah will do

**Comment by brian-c** [comment]
Subjects are limited to 1000 per set or per project or per owner?

**Comment by edpaget** [comment]
Per uploader.

**Pull request merged**